.github/workflows/check-self.yml

```diff
@@ -22,6 +22,13 @@ jobs:
       run: |
         sudo chmod a+rwx -R libs/gl-client-py
         make docker-image
+    - name: Rust Cache
+      uses: actions/cache@v2
+      with:
+        path: |
+          /tmp/gltesting/cargo
+          /tmp/gltesting/target
+        key: ${{ runner.os }}-cargo-${{ hashFiles('**/Cargo.lock') }}
```

Comment:
I never quite understood how much cache is available and what the speed to access it is. Did you find details on this (retention, size, and cache speed)? Just out of curiosity.

Reply:
I have no idea.

"If it works it works, it it doesn't it doesn't matter" is my current approach to Caching.

Reply:
Gotcha :+1: